reference/README.md

```diff
@@ -1,5 +1,11 @@
-These are bits and pieces we used in building the mask and accesories!
+# Reference
```

Comment:
These explanations on your design thinking and process are awesome.  Really great work!